src/Selenium2Driver.php

```diff
@@ -802,6 +802,19 @@ public function isVisible($xpath)
             ($this->elementIsFullyInViewport($element) || $this->elementIsPartiallyInViewport($element));
     }
 
+    /**
+     * Checks whether element is displayed located by it's XPath query.
+     *
+     * @param string $xpath
+     * @return Boolean
+     */
+    public function isDisplayed($xpath)
+    {
+        $element = $this->findElement($xpath);
```

Comment:
Please add a `throws` to the docblock for the WebDriver\Exception thrown by findElement. Please do not use the fully qualified name in the docblock as they have done in the Selenium2Driver. Instead, use either the alias or the just the class name if `Exception` does not conflict with an existing import.

- [ ] Done

Reply:
Done.